lib/screens/edit_product_screen.dart

```diff
@@ -149,6 +228,7 @@ class _EditProductScreenState extends State<EditProductScreen> {
                 focusNode: _descriptionFocusNode,
                 controller: _descriptionController,
                 validator: validatorDescription,
+                onSaved: onSavedDescription,
```

Comment:
same here. it's redundant

Reply:
I tried omitting the onSaved function, the edited product does not reflect.